app/frontend/javascript/survey_widget.js

```diff
@@ -0,0 +1,88 @@
+// This is a fairly brute-force
+// attempt to make sure that if
+// you don't want to fill in the survey
+// or already did fill it in, you
+// won't be asked again for a long time.
+
+// Despite our best efforts,
+// the survey was not registering
+// the fact that users had dismissed it
+// or already filled it in.
+
+// Note that SurveySparrow has its own
+// mechanisms in place to attempt to ensure
+// that users only fill out the survey once;
+// we just don't trust them.
+jQuery( document ).ready(function() {
+  if (jQuery('#ss_survey_widget').length != 1) {
+    return;
+  }
+  if (surveyAlreadyShown()) {
+    window.console.log("You've already clicked on the survey! Not gonna show it.")
+  }
+  else {
+    window.console.log("Launching the survey.")
+    sparrowLaunch({
+      //add custom params here
+    })
+  }
+  // Any click on the main survey div is
+  // interpreted as "user has started survey."
+  // Once they start the survey, never show
+  // this user the survey again for an entire year.
+  jQuery ('#ss_survey_widget').click(userHasStartedSurvey);
+});
+
+
+function surveyAlreadyShown() {
+  return document.cookie.match(/surveyAlreadyShown=true/) != null;
+}
+
+function userHasStartedSurvey(event) {
+  event.preventDefault();
+  var expiratonStr = new Date(new Date()
+    .setFullYear(new Date()
+    .getFullYear() + 3))
+    .toString();
+  document.cookie = "surveyAlreadyShown=true; path=/; expires=" + expiratonStr;
+  window.console.log("Set the cookie!");
+}
+
+// This is taken from the survey sparrow site.
+function sparrowLaunch(opts) {
+  var e = "ss-widget",
+    t = "script",
+    a = document,
+    r = window,
+    l = localStorage;
+  var s, n, c, rm = a.getElementById('SS_SCRIPT');
+  r.SS_WIDGET_TOKEN = "tt-5zUYbALWMcKM1wqSzFDcLi";
+  r.SS_ACCOUNT = "sciencehistory.surveysparrow.com";
+  r.SS_SURVEY_NAME = "digitial-collections-survey";
```

Comment:
You want to hard-code these in here?  Not sure what the alternative is honestly. but it seems weird. Seems at least you oughta put them in a JS `const` at top of file, with comments explaining where they come from. 

Do we plan to take out this code entirely when survey period is complete? Or are we going to leave it in for future survey use?